widgets/common/IncompatiblePointDistribution.tsx

```diff
@@ -179,4 +182,349 @@ class IncompatiblePointDistribution extends Component<IncompatiblePointDistribut
     );
   }
 }
-export default IncompatiblePointDistribution;
+
+
+type BarChartSelectorProps = {
+  selectedChart: string
+  setSelectedChart: Function
+}
+
+class BarChartSelector extends Component<BarChartSelectorProps, null> {
+
+  render () {
+    let getBlockClass  = (regionName: string) => {
```

Comment:
Minor nit - `const` is generally preferred over `let` for read-only variables like these

Reply:
This is coding style specific. Nothing wrong with this really. So leave out unless you think I actually need to do this?

Reply:
Your code is correct as is, so I don't have a problem with it if you leave it.